docs/release-notes/known-issues.md

```diff
@@ -0,0 +1,20 @@
+---
+navigation_title: "Elastic APM Java Agent"
```

Comment:
Are the "known issues" and "deprecations" pages required by some standard? Or would it be okay to keep those mentions within the single release notes page?

The reason is that we have (and want to keep) an automation to move unreleased changelog entries on release automatically into a new section within the release notes docs file for the release being made.

We don't have such a thing for the new deprectations and known-issues pages.

Reply:
@KOTungseth and @nastasha-solomon worked on the structure of the release notes section. They might be able to provide more details on the reason to split known issues and deprecations out from release notes.

Reply:
It does make sense with the merged release notes page. Also not really a blocker, we'll adjust our automation accordingly